runtime/dotnet/azurefunctions/Startup.cs

```diff
@@ -43,19 +43,17 @@ private IConfigurationRoot BuildConfiguration(string rootDirectory)
             var config = new ConfigurationBuilder();
 
             // Config precedence 1: root app.settings
-            config
-                .SetBasePath(rootDirectory)
-                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
-                .UseLuisConfigAdapter()
-                .UseLuisSettings();
+            config.SetBasePath(rootDirectory);
 
             // Config precedence 2: ComposerDialogs/settings settings which are injected by the composer publish
             // Hard code the settings path to 'ComposerDialogs' for deployment
             var configFile = Path.GetFullPath(Path.Combine(rootDirectory, @"ComposerDialogs/settings/appsettings.json"));
             config.AddJsonFile(configFile, optional: true, reloadOnChange: true);
 
-            // Config Precedence 3: Deployment specific config
-            config.AddJsonFile("appsettings.deployment.json", optional: true, reloadOnChange: true);
+            config.AddJsonFile("appsettings.json", optional: true, reloadOnChange: true);
```

Comment:
this still appears to be using 2 setttings files.

Changes in the `wenyluo/jsRuntime` branch should make this unnecessary. ALL settings are written to `settings/appsettings.json` in that branch - there should be no secondary settings file.